src/Designer/backend/src/Designer/Services/Implementation/ApplicationMetadataService.cs

```diff
@@ -38,19 +38,19 @@ public class ApplicationMetadataService : IApplicationMetadataService
         /// <param name="storageAppMetadataClient">IAltinnStorageAppMetadataClient</param>
         /// <param name="altinnGitRepositoryFactory">IAltinnGitRepository</param>
         /// <param name="httpContextAccessor">The http context accessor.</param>
-        /// <param name="giteaApiWrapper"></param>
+        /// <param name="giteaClient"></param>
```

Comment:
missing param description
```suggestion
        /// <param name="giteaClient">IGitea</param>
```